pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -486,6 +494,24 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 	return found.Name, nil
 }
 
+func setAnnotations(instance *v2vv1alpha1.VirtualMachineImport, vmSpec *kubevirtv1.VirtualMachine) {
+	annotations := instance.GetAnnotations()
+	propagate := annotations[AnnPropagate]
+	if propagate != "" {
+		annotations := vmSpec.GetAnnotations()
+		if annotations == nil {
+			annotations = map[string]string{}
+			vmSpec.SetAnnotations(annotations)
+		}
+		prop := map[string]string{}
+		err := json.Unmarshal([]byte(propagate), &prop)
+		if err != nil {
+			return
```

Comment:
pls log the error, i.e. failed to parse annotations in case of an error before returning.

Reply:
It is not critical so Info level should be good enough.